docs/designers-developers/developers/tutorials/notices/README.md

```diff
@@ -0,0 +1,77 @@
+# Displaying Notices from Your Plugin or Theme
+
+Notices are informational UI displayed near the top of admin pages. WordPress, themes, and plugins all use notices to indicate the result of an action, or to draw the user's attention to necessary information.
+
+In the Classic Editor, notices hooked onto the `admin_notices` action can render whatever HTML they'd like. In the Block Editor, notices are restricted to a more formal API.
+
+## Notices in the Classic Editor
+
+In the Classic Editor, here's an example of the "Post draft updated" notice:
+
+![Post draft updated in the Classic Editor](https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/designers-developers/developers/tutorials/notices/classic-editor-notice.png)
+
+Producing an equivalent "Post draft updated" notice would require code like this:
+
+```php
+/**
+ * Hook into the 'admin_notices' action to render
+ * a generic HTML notice.
+ */
+add_action( 'admin_notices', function() {
```

Comment:
So personally I have no issues with using closures inside actions (in fact I love them myself and almost hate that I'm commenting about this)… but I think, at least until the day core officially drops 5.2, this should be written with a full function name and all that nonsense.

Sigh.

Reply:
April 2019 per https://make.wordpress.org/core/2018/12/08/updating-the-minimum-php-version/

Reply:
FINGERS ALL THE WAY CROSSED

Reply:
Changed to a function in 07e8a600550cfc75c663b83a90cca706cd40a045